src/java.base/share/classes/java/util/zip/GZIPInputStream.java

```diff
@@ -34,8 +34,14 @@
 import java.util.Objects;
 
 /**
- * This class implements a stream filter for reading compressed data in
- * the GZIP file format.
+ * This class implements a stream filter for reading compressed data in the GZIP file format.
```

Comment:
I think you could keep this leading paragraph unchanged, including the line break. Similarly, it would be nice to keep the introduced text within 80 character lines as well.